Sources/Mustache/Context.swift

```diff
@@ -13,7 +13,7 @@
 //===----------------------------------------------------------------------===//
 
 /// Context while rendering mustache tokens
-struct MustacheContext {
+public struct MustacheContext {
```

Comment:
Is this a separate change?

Reply:
This is required for the pulicification of the `MustacheSequence` protocol.